desktop/apps/consignments2/test/reducers.js

```diff
@@ -190,11 +313,16 @@ describe('Reducers', () => {
           const middlewares = [ thunk ]
           const mockStore = configureMockStore(middlewares)
 
-          store = mockStore(initialResponse)
+          store = mockStore({
+            submissionFlow: {
+              user: { accessToken: 'foo' },
+              submission: { id: 'sub1' }
+            }
+          })
           request = sinon.stub()
           request.post = sinon.stub().returns(request)
           request.set = sinon.stub().returns(request)
-          request.send = sinon.stub().returns(stubbedToken)
+          request.send = sinon.stub().returns({ body: { token: 'i-have-access' } })
```

Comment:
Yeah, this isn't _totally_ perfect. I've found it difficult to stub multiple requests that use the same API, but for simple stuff it's good.